frontend/catalyst/compiler.py

```diff
@@ -385,10 +383,12 @@ def get_default_flags():
             "-lrt_backend",
             "-lrt_capi",
             "-lpthread",
-            f"{error_flag}",
             "-lmlir_c_runner_utils",  # required for memref.copy
         ]
 
+        if platform.system() != "Linux":  # pragma: no cover
```

Comment:
Why not checking the system == macOS instead?

Reply:
It's a long story, but the short version is this was still a draft :sweat_smile: and I wanted to run the tests on the mac mini.

```suggestion
        if platform.system() != "Darwin":  # pragma: no cover
```